src/ucp/core/ucp_ep.c

```diff
@@ -261,6 +261,9 @@ static UCS_F_NOINLINE void ucp_ep_remove_progress_callbacks(ucp_ep_h ep)
     ucs_callbackq_remove_if(&ep->worker->uct->progress_q,
                             ucp_wireup_msg_ack_cb_pred, ep);
 
+    ucs_callbackq_remove_if(&worker->uct->progress_q,
```

Comment:
Maybe unite all these remove_if into one predicate to reduce amount of code?

Reply:
done